### PR TITLE
add package talib

### PR DESCRIPTION
**Pull request by ifurther**
#5522

### CODE REVIEW

File: packages/talib/build.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+TERMUX_PKG_DESCRIPTION="Technical analysis library with indicators like ADX."
+TERMUX_PKG_LICENSE="BSD"
+TERMUX_PKG_VERSION=0.4.0
+TERMUX_PKG_SRCURL=http://prdownloads.sourceforge.net/ta-lib/ta-lib-$TERMUX_PKG_VERSION-src.tar.gz
+TERMUX_PKG_SHA256=9ff41efcb1c011a4b4b6dfc91610b06e39b1d7973ed5d4dee55029a0ac4dc651
+#TERMUX_PKG_DEPENDS=""
+TERMUX_PKG_BUILD_IN_SRC=true
+
+termux_step_pre_configure() {
+	curl https://sourceforge.net/p/ta-lib/code/HEAD/tree/tags/release-0-4-0/ta-lib/LICENSE.TXT?format=raw -o $TERMUX_PKG_SRCDIR/LICENSE
```
Comment (landfillbaby) [inline packages/talib/build.sh:11]: use `${TERMUX_PKG_VERSION//./-}` instead of `0-4-0`

Comment (ifurther) [reply packages/talib/build.sh]: OK, I fixed it.

File: packages/talib/build.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
+TERMUX_PKG_HOMEPAGE=https://sourceforge.net/projects/ta-lib/
+TERMUX_PKG_DESCRIPTION="Technical analysis library with indicators like ADX."
+TERMUX_PKG_LICENSE="BSD"
+TERMUX_PKG_VERSION=0.4.0
+TERMUX_PKG_SRCURL=http://prdownloads.sourceforge.net/ta-lib/ta-lib-$TERMUX_PKG_VERSION-src.tar.gz
```
Comment (Grimler91) [inline packages/talib/build.sh:5]: Please use https instead of http

Comment (ifurther) [reply packages/talib/build.sh]: Done

**Pull request merged**